nearcore/benches/store.rs

```diff
@@ -0,0 +1,78 @@
+#[macro_use]
+extern crate bencher;
+
+use bencher::Bencher;
+use near_chain::{ChainStore, ChainStoreAccess, RuntimeAdapter};
+use near_chain_configs::GenesisValidationMode;
+use near_logger_utils::init_integration_logger;
+use near_primitives::types::StateRoot;
+use near_store::{create_store, TrieIterator};
+use nearcore::{get_default_home, get_store_path, load_config, NightshadeRuntime};
+use std::time::{Duration, Instant};
+
+/// Read `TrieItem`s - nodes containing values - using Trie iterator, stop when `num_trie_items` items were read.
+/// TODO: make separate bench runs independent. As of 18/01/2022, first run gives speed of 50 items per second, and all
+/// next runs give ~ 30k items per second.
+fn read_trie_items(bench: &mut Bencher, num_trie_items: usize, shard_id: usize) {
+    init_integration_logger();
+    let home_dir = get_default_home();
+    let near_config = load_config(&home_dir, GenesisValidationMode::UnsafeFast);
+
+    bench.iter(move || {
+        tracing::info!(target: "neard", "{:?}", home_dir);
+        let store = create_store(&get_store_path(&home_dir));
```

Comment:
can you add the option to use the readonly store? (I wonder if it is faster then -as it doesn't do the compactions, right?)

can be done in separate pr

Reply:
My experiment showed that performance is the same.